### ISSUE TITLE
Not finding blockchain.com password even though i know it is the right password

### ISSUE THREAD

**Issue by Hurleybim** [3rdIteration/btcrecover#464]
Hi,
I have a problem with one wallet. I lost the blockchain.com password and a seed phrase was never created, but I have a fair idea of what the password could be. I used btcrecover to get the wallet.aes.json file and run a few token lists on it to no success. However, I recently tried another blockchain account of mine and I was able to access it with password. Out of curiosity, I ran btcrecover on the second wallet with token lists and password lists which should both have found the password - as I entered it in both lists - but btcrecover finished the search without finding the password. That makes me think that all of the searches I have run on the first wallet were wasted as btcrecover seems not to acknowledge the correct password even when it is in the lists. Is anyone familiar with this problem or have any recommendations? Both wallets were created in 2015/16. I would greatly appreciate any help on this. I would love to get btcrecover to register the right password on the second wallet so I know it works and can be confident in running the searches on the first wallet.

**Comment by 3rdIteration** [comment]
Did you check the possible passwords.log? If nothing appeared in there, are you able to share the sample wallet file?

**Comment by Hurleybim** [comment]
> Did you check the possible passwords.log? If nothing appeared in there, are you able to share the sample wallet file?

Yes I did check possible passwords log.
Here is the last entry there which i do not understand as i ran it on an entirely different wallet than the test wallet

"2024-04-17 10:48:01 New Recovery Started...
This file contains passwords and blocks from passwords which `may` not exactly match those that BTCRecover searches for by default. 

Examples of successfully decrypted blocks will not just be random characters, some examples of what correctly decryped blocks logs look like are:

Possible Password ==>btcr-test-password<== in Decrypted Block ==>{
"guid" : "9bb<==
Possible Password ==>testblockchain<== in Decrypted Block ==>{"address_book":<==
Possible Password ==>btcr-test-password<== in Decrypted Block ==>{"tx_notes":{},"
Possible Password ==>Testing123!<== in Decrypted Block ==>{"double_encrypt<==

Note: The markers ==> and <== are not part of either your password or the decrypted block...

If the password works and was not correctly found, or your wallet detects a false positive, please report the decrypted block data at https://github.com/3rdIteration/btcrecover/issues/"

I am attaching the wallet json file for the wallet whose password i know works but btcrecover isnt recognising as correct
[wallet.aes.json](https://github.com/3rdIteration/btcrecover/files/15023366/wallet.aes.json)

Any help with this would be GREATLY appreciated and i will definitely leave a generous tip if i can access the other wallet.

**Comment by 3rdIteration** [comment]
It does t look like the possible passwords file has anything other than what is in there by default.

Are you able to share the test wallet and known password?

**Comment by 3rdIteration** [comment]
Ah yea the problem with the test wallet there is that you didn't download it properly.

You need to only include the payload and also get rid of all the double slashes in there so that it is formatted the same as the examples in the BTCRecover repo. If you download the wallet file with Chrome then this is done for you ..

**Comment by Hurleybim** [comment]
ok i got the response copied the wallet using chrome web developer tools and removed the double slashes (they were still there). They were still there. I also removed some of the single slashes. I have created both a password list and tokenlist file both of which have the correct password. I run btcrecover on the wallet file and still it hasnt shown the password as the correct one. I will send my new wallet file created after your suggestion along with the password list and tokenlist. The password is -> Dasthu27. <-
[passlist.txt](https://github.com/3rdIteration/btcrecover/files/15026382/passlist.txt)
[tokens.txt](https://github.com/3rdIteration/btcrecover/files/15026383/tokens.txt)
[wallet.aes.json](https://github.com/3rdIteration/btcrecover/files/15026384/wallet.aes.json)
